Legion/VaultOfTheWardens/Tormentorum.lua

```diff
@@ -129,4 +132,28 @@ do
 			self:Message(args.spellId, "Personal", "Alert", CL.underyou:format(args.spellName))
 		end
 	end
+end
+
+do
+	local prev = 0
+	local list = mod:NewTargetList()
+	function mod:FleshtoStoneApplied(args)
+		local amount = args.amount or 1
+		if amount < 7 then
+			return
+		end
+
+		local t = GetTime()   -- 2 of these adds spawn, they will finish their casts
+		if t - prev < 5 then  -- roughly at the same time, throttling to avoid being spammy
+			return
+		end
+		prev = t
+
+		local count = #list+1
+		list[count] = args.destName
+
+		if count == 1 then
+			self:ScheduleTimer("TargetMessage", 0.3, format("%s (7+)", self:SpellName(args.spellId)), list, "Urgent", self:Dispeller("magic") and "Warning", nil, nil, true)

```

Comment:
No need for that `self:SpellName` call, should be `args.spellName` instead.

Reply:
Wrong arguments for `TargetMessage`. `key` is missing, `text` is in the wrong place.